### ISSUE TITLE
Steroid-responsive anemia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5679]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
see https://pubmed.ncbi.nlm.nih.gov/28280134/

**Parent term (use hpo.jax.org/app)**
Anemia

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://pubmed.ncbi.nlm.nih.gov/28280134/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term
